script.kodi.windows.update/default.py

```diff
@@ -0,0 +1,249 @@
+#     Copyright (C) 2020 Team-Kodi
```

Comment:
spdx headers would be nice, look at the kodi source code for examples

Reply:
Can you send me a link with the available spdx headers?

Reply:
Any CPP file will do, see for example https://github.com/xbmc/xbmc/blob/master/xbmc/AppInboundProtocol.h#L1-L7
Make sure you adapt to your LICENSE path instead of a direct copy-paste (`LICENSES/README.md`)